Userland/Libraries/LibWeb/CSS/Serialize.cpp

```diff
@@ -117,6 +117,28 @@ void serialize_a_url(StringBuilder& builder, StringView url)
     builder.append(')');
 }
 
+// https://www.w3.org/TR/cssom-1/#serialize-a-local
+void serialize_a_local(StringBuilder& builder, StringView path)
+{
+    // To serialize a LOCAL means to create a string represented by "local(",
+    // followed by the serialization of the LOCAL as a string, followed by ")".
+    builder.append("local("sv);
+    serialize_a_string(builder, path.to_string());
+    builder.append(')');
+}
+
+// NOTE: No spec currently exists for serializing a <'unicode-range'>.
+void serialize_unicode_ranges(StringBuilder& builder, Vector<UnicodeRange> unicode_ranges)
```

Comment:
```suggestion
void serialize_unicode_ranges(StringBuilder& builder, Vector<UnicodeRange> const& unicode_ranges)
```
Please pass Vectors by reference, otherwise the whole Vector gets copied.

Reply:
Apologies, I need to read up more on pointers in C++ and in Serenity.